src/App.tsx

```diff
@@ -7,17 +7,19 @@ import { StyleSheet, Text, View } from 'react-native';
 import Ionicons from '@expo/vector-icons/Ionicons';
 
 import Contributors from './Contributors';
+import ButtonComponent from './components/ButtonComponent';
 
 const Tab = createBottomTabNavigator();
 
 const HomeScreen = () => {
   return (
     <View style={styles.container}>
       <Text>Swarm Beekeeping App</Text>
+      <ButtonComponent text="Get Started" handleClick={() => console.log('clicked')} />
```

Comment:
We can define the strings elsewhere after this PR when you start using your component